packages/story-editor/src/utils/usePageAsCanvas.js

```diff
@@ -95,9 +98,18 @@ function usePageAsCanvas() {
         if (!pageCanvasPromise && fullbleedContainer) {
           import(/* webpackChunkName: "chunk-html-to-image" */ 'html-to-image')
             .then((htmlToImage) => {
+              const hasSingleSelection = selectedElementIds.length === 1;
+              const filterSelectedElement =
+                skipSelectedElement && hasSingleSelection;
+              const filter = (node) => {
+                // exclude selected element from generated image to prevent interfereing with contrast calculation. true includes and false excludes
```

Comment:
```suggestion
                // Exclude selected element from generated image to prevent interfering with contrast calculation: `true` includes and `false` excludes.
```